app/views/survey/manage.html.erb

```diff
@@ -0,0 +1,22 @@
+<h4>Create a question</h4>
+<%= javascript_include_tag 'manageSurvey' %>
+<form action="/survey/manage" method="post" id="newQuestion">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="question">Enter a question</label>
+  <input type="text" name="question" />
+  <hr />
+  <p>Now add some possible answers. You can have one to four answers.</p>
+  <label for="answer1">Answer 1:</label>
+  <input type="text" name="answer1" />
+  <br />
+  <label for="answer2">Answer 2:</label>
+  <input type="text" name="answer2" />
+  <br />
+  <label for="answer3">Answer 3:</label>
+  <input type="text" name="answer3" />
+  <br />
+  <label for="answer4">Answer 4:</label>
+  <input type="text" name="answer4" />
+  <br/>
+  <button type="button" id="submitNewQuestion" onclick="submitQuestion()">Submit</button>
```

Comment:
There are things that you can do with rails submissions and alerts that will do user feedback.